src/common/quassel.h

```diff
@@ -142,6 +142,7 @@ class COMMON_EXPORT Quassel : public QObject, public Singleton<Quassel>
         EcdsaCertfpKeys,      ///< ECDSA keys for CertFP in identities
         LongMessageId,        ///< 64-bit IDs for messages
         SyncedCoreInfo,       ///< CoreInfo dynamically updated using signals
+        LoadBacklogForwards,  ///< Allow loading backlog in reverse
```

Comment:
Very minor - it might be more intuitive to explain in terms of time, to avoid juxtaposing "forwards" and "reverse" :slightly_smiling_face: 

```suggestion
        LoadBacklogForwards,  ///< Allow loading backlog in ascending order, old to new
```